src/app.js

```diff
@@ -10,8 +10,10 @@ require('./config/passport');
 require('dotenv').config();
 const { dbClient } = require('./database/index');
 const usersRouter = require('./modules/user/routes');
+
 const accommodationRouter = require('./modules/accommodations/routes');
 const swaggerConfig = require('../swagger.json');
+const bookingRoute = require('./modules/Booking/routes');
```

Comment:
since there will be multiple routes you might consider renaming the variable as `bookingRoutes` since this increases readability and show that there are multiple routes for booking there.